### ISSUE TITLE
Show photo page of image behind topo

### ISSUE THREAD

**Issue by quaestor** [theCrag/website#2943]
**What happened?**

There is no way to show the original image behind a topo. This would sometimes be useful to add new routes, because in the small version shown in the topo editor, you sometimes cannot see enough details to spot where the new line is (e.g. bolts, pockets). If you didn't create the topo yourself, you might not have another reference picture. This also relates to #2353, because uploading your own photo (so you have a full-size reference) is cumbersome, as you need to redraw all the routes, even if you might have a picture from a similar angle.

**What you expected:**

Add a button to the topo dropdown to show the full-size photo.

**Comment by brendanheywood** [comment]
There is a few moving parts to this:

a) I will 'soonish' be polishing off a bug fix for the zooming where the images are gpu scaled in some browsers and so are blurred #1911 - hopefully this will cover 90% of this class of issue.
a) I will eventually be making a much larger improvement to the topo tool so you can zoom in as much as you want #2871. This will also enable 'mega topos' and this should fix the remaining 10% of user cases where you still can't see enough detail

Also the direction I'd like to take here is that all topos conceptually are photos, ie if you upload a topo then it also appears in the list of photos, and in the photo detail page it also shows that it has been used as a topo, similar to how you can see it has been used as a webcover. This will avoid a lot of confusion and fix #346. And also you turn any photo into a topo (bug #2693, this used to work). So you would be able to link from the topo edit page to it's photo page.

As a technical interim workaround, if you right click and inspect the topo html, open the image in a new tab and then remove the cropping dimensions you will get the original photo in it's full size.

**Comment by quaestor** [comment]
Ah, I didn't know you could just remove the cropping dimensinos in the URL, that's helpful, thanks! Also looking forward to those changes you mentioned.

**Comment by Mdemaillard** [comment]
see similar for profile avatar #1472 2943

**Comment by rouletout** [comment]
I don't think that this is still an issue as we serve high res now for topos? Closing, please re-open if you think this is still open.

**Comment by quaestor** [comment]
@rouletout this is still open. Take for example https://www.thecrag.com/en/climbing/germany/frankenjura/wolkensteiner-wand/topos. How would you access the photo page/full size picture behind that topo? There is currently no way besides opening the developer console and looking for the URL in the source code. This hasn't changed just because the topos are now displayed in a larger size by default (they have been stored in high resolution for years and I've been using Brendan's trick since 2017).

Brendan's idea seems to be the most plausible solution:

> Also the direction I'd like to take here is that all topos conceptually are photos, ie if you upload a topo then it also appears in the list of photos, and in the photo detail page it also shows that it has been used as a topo, similar to how you can see it has been used as a webcover.

I don't think I'm allowed to reopen tickets, so would you please do that?

**Comment by rouletout** [comment]
ok, at least you can now first load a foto and then use it for a topo....

**Comment by nicHoch** [comment]
<img width="2051" height="820" alt="Image" src="https://github.com/user-attachments/assets/bdb8e927-f716-4aec-afb1-0ee69ca9f93b" />

on the topo edit page there will be two new link options: to the background image and the dedicated topo page
